lab_5_scrapper/scrapper.py

```diff
@@ -90,25 +213,45 @@ def __init__(self, config: Config) -> None:
         """
         Initializes an instance of the Crawler class
         """
-        pass
+        self._seed_urls = config.get_seed_urls()
+        self._config = config
+        self.urls = []
```

Comment:
should it be protected?

Reply:
I'm not sure. The lab description says: Each instance should also have an additional attribute `self.urls`, initialized with empty list.

Reply:
agree